cmds/core/uniq/uniq.go

```diff
@@ -48,7 +48,7 @@ var (
 // var fnum = flag.Int("f", 0, "ignore num fields from beginning of line")
 // var cnum = flag.Int("cn", 0, "ignore num characters from beginning of line")
 
-func uniq(f *os.File) {
+func uniq(f *os.File, w io.Writer) {
```

Comment:
I am wonder if we can make this an io.Reader. butio.NewReader will wrap it just fine even if it's an io.Reader.